config/v1alpha1/types_cluster_monitoring.go

```diff
@@ -416,6 +427,549 @@ type MetricsServerConfig struct {
+	// across restarts.
+	// +optional
+	VolumeClaimTemplate VolumeClaimConfig `json:"volumeClaimTemplate,omitempty,omitzero"`
+}
+
+
+type AlertmanagerScheme string
+
+const (
+	AlertmanagerSchemeHTTP  AlertmanagerScheme = "HTTP"
+	AlertmanagerSchemeHTTPS AlertmanagerScheme = "HTTPS"
+)
+
+// AdditionalAlertmanagerConfig represents configuration for additional Alertmanager instances.
+// The `AdditionalAlertmanagerConfig` resource defines settings for how a
+// component communicates with additional Alertmanager instances.
+type AdditionalAlertmanagerConfig struct {
+	// name is a unique identifier for this Alertmanager configuration entry.
+	// This field is used as the map key for server-side apply, allowing multiple
+	// controllers (e.g., ACM, cluster admins) to independently manage their own entries.
+	// The name must be a valid DNS subdomain (RFC 1123): lowercase alphanumeric characters,
+	// hyphens, or periods, and must start and end with an alphanumeric character.
+	// Maximum length is 253 characters.
+	// +kubebuilder:validation:MaxLength=253
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:XValidation:rule="self.matches('^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$')",message="must be a valid DNS subdomain: lowercase alphanumeric, hyphens, or periods"
+	// +required
+	Name string `json:"name,omitempty"`
+	// authorization configures the authentication method for Alertmanager connections.
+	// Supports bearer token authentication. When omitted, no authentication is used.
+	// +optional
+	Authorization AuthorizationConfig `json:"authorization,omitempty,omitzero"`
+	// pathPrefix defines an optional URL path prefix to prepend to the Alertmanager API endpoints.
+	// For example, if your Alertmanager is behind a reverse proxy at "/alertmanager/",
+	// set this to "/alertmanager" so requests go to "/alertmanager/api/v1/alerts" instead of "/api/v1/alerts".
+	// This is commonly needed when Alertmanager is deployed behind ingress controllers or load balancers.
+	// Must start with "/" and not end with "/" (unless it is the root path "/").
+	// Must not contain query strings ("?") or fragments ("#").
+	// +kubebuilder:validation:MaxLength=255
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:XValidation:rule="self.startsWith('/')",message="pathPrefix must start with '/'"
+	// +kubebuilder:validation:XValidation:rule="!self.endsWith('/') || self == '/'",message="pathPrefix must not end with '/' (except for root '/')"
+	// +kubebuilder:validation:XValidation:rule="!self.contains('?') && !self.contains('#')",message="pathPrefix must not contain '?' or '#'"
+	// +optional
+	PathPrefix string `json:"pathPrefix,omitempty"`
+	// scheme defines the URL scheme to use when communicating with Alertmanager
+	// instances.
+	// Possible values are `HTTP` or `HTTPS`.
+	// When omitted, defaults to `HTTP`.
+	// +kubebuilder:validation:Enum=HTTP;HTTPS
```

Comment:
If you use a type alias, we generally prefer that this marker is placed on the type alias.